packages/components/organisms/f-checkout/src/components/UserNote.vue

```diff
@@ -1,30 +1,31 @@
 <template>
     <div :class="$style['c-userNote']">
-        <label
-            for="userNote"
-            :class="$style['c-userNote-title']">
-            {{ $t('userNote.title') }}
-        </label>
-        <div :class="$style['c-userNote-content']">
-            {{ $t('userNote.text') }}
-        </div>
-        <textarea
-            id="userNote"
-            name="notePlaceholder"
-            cols="30"
-            rows="10"
-            maxlength="200"
-            :class="$style['c-userNote-textArea']"
+        <form-field
+            :label-text="$t('userNote.title')"
+            input-type="textarea"
             :placeholder="$t('userNote.placeholder')"
-            @input="updateUserNote($event.target.value)" />
+            :cols="30"
+            :rows="7"
+            :maxlength="200"
+            :show-extra-label-text="true"
```

Comment:
You could omit the "true" part here and the same for `has-input-description` below

```suggestion
            show-extra-label-text
```